crates/admin/protobuf/cluster_ctrl_svc.proto

```diff
@@ -99,7 +99,17 @@ message ChainExtension {
   optional uint32 segment_index = 2;
   // check `ProviderKind` for possible values.
   string provider = 4;
+
+  // [deprecated] We return an error if this
+  // field is set.
```

Comment:
Nice! In the future we could require an explicit version to be passed in, and then we can actually drop this.